### PR TITLE
Adds 4.9.29 release notes

### PR DESCRIPTION
**Pull request by darshan-nagaraj**
No BZ.

To be merged on 04-20. I will send a reminder today when the erratas are shipped live.

OCP version: **only Enterprise-4.9**

Direct Doc Preview Link: https://deploy-preview-44728--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-29

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -2756,3 +2756,27 @@ link:https://access.redhat.com/solutions/6906441[{product-title} 4.9.28 containe
...
+
+* Previously, `oc adm must gather` fell back to the `oc adm inspect` command when the specified image could not run. Consequently, it was difficult to understand information from the logs when the fall back happened. With this update, the logs are improved to make it explicit when a fall back inspection is performed. As a result, the output of `oc adm must gather` is easier to understand. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2051944[*BZ#2051944*])
+
+* Previously, hardcoded BusyBox image from the Docker was used for the scorecard. Consequently, the Docker's new rate limit caused periodic failures when running the scorecard. This fix recommends to use the Universal Base Image (UBI) `registry.access.redhat.com/ubi8/ubi:8.4` for the scorecard, resulting no failures due to rate limiting. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2064408[*BZ#2064408*])
```
Comment (gaurav-nelson) [inline release_notes/ocp-4-9-release-notes.adoc:2777]:
```suggestion
* Previously, hard-coded BusyBox image from the Docker was used for the scorecard. Consequently, the Docker's new rate limit caused periodic failures when running the scorecard. This fix recommends to use the Universal Base Image (UBI) `registry.access.redhat.com/ubi8/ubi:8.4` for the scorecard, resulting no failures due to rate limiting. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2064408[*BZ#2064408*])
```

Comment (gaurav-nelson) [reply release_notes/ocp-4-9-release-notes.adoc:2777]: https://redhat-documentation.github.io/supplementary-style-guide/#hard-coded

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -2756,3 +2756,27 @@ link:https://access.redhat.com/solutions/6906441[{product-title} 4.9.28 containe
...
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6934361[{product-title} 4.9.29 container image list]
+
+[id="ocp-4-9-29-bug-fixes"]
+==== Bug fixes
+* Previously, the Local Storage Operator (LSO) added an `OwnerReference` object to the persistent volumes (PV) it created, which sometimes caused an issue where a delete request for a PV could leave the PV in the `terminating` state while still attached to the pod. With this update, the LSO no longer creates an `OwnerReference` object and cluster administrators are now able to manually delete any unused PVs after a node is removed from the cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2070617[*BZ#2070617*])
```
Comment (gaurav-nelson) [inline release_notes/ocp-4-9-release-notes.adoc:2773]: Previously, the Local Storage Operator (LSO) added an `OwnerReference` object to the persistent volumes (PV) it created, which sometimes caused an issue where a delete request for a PV could leave the PV in the `terminating` state while still attached to the pod. With this update, the LSO no longer creates an `OwnerReference` object, and cluster administrators can delete unused PVs after a node is removed from the cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2070617[*BZ#2070617*])

Comment (gaurav-nelson) [inline release_notes/ocp-4-9-release-notes.adoc:2773]:
```suggestion
Previously, the Local Storage Operator (LSO) added an `OwnerReference` object to the persistent volumes (PV) it created, which sometimes caused an issue where a delete request for a PV could leave the PV in the `terminating` state while still attached to the pod. With this update, the LSO no longer creates an `OwnerReference` object, and cluster administrators can delete unused PVs after a node is removed from the cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2070617[*BZ#2070617*])
```

File: release_notes/ocp-4-9-release-notes.adoc [hunk 3]
```diff
@@ -2756,3 +2756,27 @@ link:https://access.redhat.com/solutions/6906441[{product-title} 4.9.28 containe
...
+
+* Previously, `oc adm must gather` fell back to the `oc adm inspect` command when the specified image could not run. Consequently, it was difficult to understand information from the logs when the fall back happened. With this update, the logs are improved to make it explicit when a fall back inspection is performed. As a result, the output of `oc adm must gather` is easier to understand. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2051944[*BZ#2051944*])
```
Comment (gaurav-nelson) [inline release_notes/ocp-4-9-release-notes.adoc:2775]: **issue:** Change `fall back` to `fallback`.

Comment (gaurav-nelson) [reply release_notes/ocp-4-9-release-notes.adoc:2775]: **suggestion**: Maybe
```
Previously, `oc adm must gather` fell back to the `oc adm inspect` command when the specified image could not run. 
Consequently, it was not easy to understand when the fallback happened by checking the logs. 
This update includes improved logs that clearly show the information when {ocp} performs fallback inspections. 
As a result, the output of `oc adm must gather` is easier to understand. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2051944[*BZ#2051944*])
```

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (darshan-nagaraj) [reply release_notes/ocp-4-9-release-notes.adoc:2777]: Fixed.

File: release_notes/ocp-4-9-release-notes.adoc (hunk 2 above)
Comment (darshan-nagaraj) [reply release_notes/ocp-4-9-release-notes.adoc:2773]: Done!

File: release_notes/ocp-4-9-release-notes.adoc (hunk 3 above)
Comment (darshan-nagaraj) [reply release_notes/ocp-4-9-release-notes.adoc:2775]: @gaurav-nelson This doc-text has been already published in other versions of OCP. Just to be consistent with other versions, I'm not considering your suggestion.

**Pull request merged**